src/v2/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -90,7 +90,7 @@ export class ArtworkApp extends React.Component<Props> {
       }
 
       const clientSideRoutingReferrer =
-        window.analytics.__artsyClientSideRoutingReferrer
+        window.analytics?.__artsyClientSideRoutingReferrer
```

Comment:
We def don't want to make this optional

Reply:
This is a work around to get the app to load locally, and I don't intend to commit these changes (notice the PR is still a Draft). I don't know what was causing it to be null, I simply needed the `user/payments2` route to load. Happy to look into this later but for now I'm just trying to wrap up this work in time to demo it!

Reply:
The thing is, there's no `FIXME` here or any other indication that this is temporary. When `?` gets added its very easy to overlook in PR review and commit to master, which then makes the code more brittle. 

> Happy to look into this later but for now I'm just trying to wrap up this work in time to demo it!

If demoing it means: committing, merging, and deploying to staging, we should slow down a bit, because there's another bug in a critical application code path that needs to be fixed first before anything can be deployed (outside of a review app)